### PR TITLE
Releases: new developers/releases page 

### PR DESCRIPTION
**Pull request by AlbertCarreras**
#### What changed?

Releases: new Releases page 

It consolidates information previously present in Tooling, Development process and Installation.

#### Why?

Improve visibility of codemod tool

### Links

[GESTALT-4213](https://jira.pinadmin.com/browse/GESTALT-4213)

### CODE REVIEW

File: docs/pages/get_started/developers/release_codemods.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import Page from '../../../docs-components/Page.js';
+
+export default function EslintPluginPage(): Node {
+  return (
+    <Page title="Release codemods (web)">
+      <PageHeader
+        name="Release codemods (web)"
+        description={`Run codemods to automate required code changesrelated to major releases (breaking changes) in Gestalt.
```
Comment (dangerismycat) [inline docs/pages/get_started/developers/release_codemods.js:12]: typo: missing space between "changes" and "related"

File: docs/pages/get_started/developers/release_codemods.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+Every major release in the component library comes with a codemod to facilitate the upgrade of the Gestalt dependency. Some codemods are custom built for each upgrade while others take advantage of generic codemods that only require running a codemod command with arguments.
+
+When releases include breaking changes, the specific codemod command is included in the pull request.`}
+        type="guidelines"
+      />
+      <MainSection
+        name="Generic codemods"
+        description={`Generic codemods for common tasks like renaming components or props can be found in this [directory](https://github.com/pinterest/gestalt/tree/master/packages/gestalt-codemods/generic-codemods).
+
+        throwErrorIfSpreadProps({
```
Comment (dangerismycat) [inline docs/pages/get_started/developers/release_codemods.js:23]: this looks like a typo?

File: docs/pages/get_started/developers/release_codemods.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+Example:
+~~~bash
+yarn codemod renameComponent ~/code/pinboard/webapp
+--componentName=Flyout
+--nextComponentName=Popover
+~~~
+
+See codemod code: [renameComponent](https://github.com/pinterest/gestalt/blob/master/packages/gestalt-codemods/generic-codemods/renameComponent.js)
+      `}
+        ></MainSection.Subsection>
```
Comment (dangerismycat) [inline docs/pages/get_started/developers/release_codemods.js:68]: this should be self-closing — this is why the preview build failed

File: docs/pages/get_started/developers/release_codemods.js [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+
+Error messages include the location (file path) and the code line of the node that caused the error to facilitate addressing the corresponding issue.`}
+      >
+        <MainSection.Subsection
+          title="renameComponent"
+          description={`
+Codemod to **RENAME** Gestalt **components**
+
+Example: transforms first code (top) into second code (bottom)
+~~~bash
```
Comment (dangerismycat) [inline docs/pages/get_started/developers/release_codemods.js:35]: this should be `js`, no? (here and throughout the examples)

Comment (AlbertCarreras) [reply docs/pages/get_started/developers/release_codemods.js]: YEp, didnt know that

File: docs/pages/get_started/developers/releases.js [hunk 5]
```diff
@@ -0,0 +1,399 @@
+// @flow strict
+import { type Node } from 'react';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import Page from '../../../docs-components/Page.js';
+import Card from '../../../docs-components/Card.js';
+import { Text, Flex, Heading, Link } from 'gestalt';
+import Markdown from '../../../docs-components/Markdown.js';
+
+export default function EslintPluginPage(): Node {
```
Comment (dangerismycat) [inline docs/pages/get_started/developers/releases.js:10]: nittest of nits: this function name isn't relevant 😛

**Review by dangerismycat — APPROVED** [review]
Sweet, thanks for consolidating all the existing info for this page!

**Pull request merged**